WordPress/Classes/ViewRelated/MeViewController.swift

```diff
@@ -0,0 +1,271 @@
+import UIKit
+import WordPressShared
+
+class MeViewController: UITableViewController, UIViewControllerRestoration {
+    static let restorationIdentifier = "WPMeRestorationID"
+    var handler: ImmuTableViewHandler!
+
+    static func viewControllerWithRestorationIdentifierPath(identifierComponents: [AnyObject], coder: NSCoder) -> UIViewController? {
+        return self.init()
+    }
+
+    // MARK: - Table View Controller
+
+    override init(style: UITableViewStyle) {
+        super.init(style: style)
+        self.restorationIdentifier = self.dynamicType.restorationIdentifier
+        self.restorationClass = self.dynamicType
+    }
+
+    required convenience init() {
+        self.init(style: .Grouped)
+        let notificationCenter = NSNotificationCenter.defaultCenter()
+        notificationCenter.addObserver(self, selector: "refreshModelWithNotification:", name: WPAccountDefaultWordPressComAccountChangedNotification, object: nil)
+        notificationCenter.addObserver(self, selector: "refreshModelWithNotification:", name: HelpshiftUnreadCountUpdatedNotification, object: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    deinit {
+        NSNotificationCenter.defaultCenter().removeObserver(self)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.title = NSLocalizedString("Me", comment: "Me page title")
+
+        ImmuTable.registerRows([
+            NavigationItemRow.self,
+            BadgeNavigationItemRow.self,
+            ButtonRow.self,
+            DestructiveButtonRow.self
+            ], tableView: self.tableView)
+
+        handler = ImmuTableViewHandler(takeOver: self)
```

Comment:
The reason I used `takeOver` is because I wanted to hint that just by initialising it would "take over" being the data source and delegate. An alternative I've been considering is to have a `init()` without arguments and then `takeOver(controller)`, but then you'd need two calls each time you want to use it.